src/Web/Telegram/API/Bot/API.hs

```diff
@@ -68,182 +73,179 @@ newtype Token = Token Text
 telegramBaseUrl :: BaseUrl
 telegramBaseUrl = BaseUrl Https "api.telegram.org" 443 ""
 
--- | Type for token
-type TelegramToken = Capture ":token" Token
-
 -- | Telegram Bot API
 type TelegramBotAPI =
```

Comment:
Here is what I'm talking about:

I consider this bad:
```haskell
sendMessage token alice "hello"
sendMessage token bob "hello"
sendMessage token charlie "hello"
```

I consider this good:
```haskell
runClient token $ do
    sendMessage alice "hello"
    sendMessage bob "hello"
    sendMessage charlie "hello"
```

What do you think?